test/specs/rules/validate-indentation.js

```diff
@@ -156,6 +156,23 @@ describe('rules/validate-indentation', function() {
         ]);
     });
 
+    it('should not report errors for using tabs when validateIndentation is a number', function() {
+        checker.configure({ validateIndentation: 4 });
```

Comment:
The problem I was having was when you use a global .jscsrc file that has validateIndentation: 4 in it, but then you edit a file that uses tabs. Jscs will report incorrect indentation on every single line even though the indentation looks correct. Also there's the rules disallowTabs and disallowMixedSpacesAndTabs to take on related problems.

Also is the test not passing or is my code not working? I just tried it with and without and it behaved as expected.